configs/ttalps_analysis/ttalps_plotter_config.py

```diff
@@ -47,23 +49,25 @@
 legend_width = 0.17 if show_ratio_plots else 0.20
 legend_min_x = 0.45
 legend_max_x = 0.83
+# legend_max_x = 0.80
```

Comment:
Some of the things commented out in this file we should just remove (and keep one version).